tests/Integration/Behaviour/Features/Scenario/Product/Combination/update_combination_from_listing.feature

```diff
@@ -32,33 +32,33 @@ Feature: Update product combination from listing in Back Office (BO)
-    When I update combination "product1Blue" from list with following values:
-      | impact on price       | 5      |
-      | quantity              | 10     |
-      | is default            | true   |
-      | combination reference | test_1 |
+    When I update combination "product1SBlue" from list with following values:
+      | impact on price | 5      |
+      | quantity        | 10     |
+      | is default      | true   |
+      | reference       | test_1 |
     And I update combination "product1MWhite" from list with following values:
-      | impact on price       | -4.99  |
-      | quantity              | 9      |
-      | is default            | false  |
-      | combination reference | test 2 |
+      | impact on price | -4.99  |
+      | quantity        | 9      |
+      | is default      | false  |
+      | reference       | test 2 |
     And I update combination "product1MBlack" from list with following values:
       | quantity | 50 |
-    Then product product1 should have following list of combinations:
-      | reference      | combination name        | combination reference | attributes           | impact on price | final price | quantity | is default |
-      | product1SWhite | Size - S, Color - White |                       | [Size:S,Color:White] | 0               | 100.99      | 0        | false      |
-      | product1SBlack | Size - S, Color - Black |                       | [Size:S,Color:Black] | 0               | 100.99      | 0        | false      |
-      | product1Blue   | Size - S, Color - Blue  | test_1                | [Size:S,Color:Blue]  | 5               | 105.99      | 10       | true       |
-      | product1MWhite | Size - M, Color - White | test 2                | [Size:M,Color:White] | -4.99           | 96          | 9        | false      |
-      | product1MBlack | Size - M, Color - Black |                       | [Size:M,Color:Black] | 0               | 100.99      | 50       | false      |
-      | product1MBlue  | Size - M, Color - Blue  |                       | [Size:M,Color:Blue]  | 0               | 100.99      | 0        | false      |
-    And product product1 default combination should be "product1Blue"
+    Then product "product1" should have following combinations:
+      | id reference   | combination name        | reference | attributes           | impact on price | final price | quantity | is default |
+      | product1SWhite | Size - S, Color - White |           | [Size:S,Color:White] | 0               | 100.99      | 0        | false      |
+      | product1SBlack | Size - S, Color - Black |           | [Size:S,Color:Black] | 0               | 100.99      | 0        | false      |
+      | product1SBlue  | Size - S, Color - Blue  | test_1    | [Size:S,Color:Blue]  | 5               | 100.99      | 10       | true       |
+      | product1MWhite | Size - M, Color - White | test 2    | [Size:M,Color:White] | -4.99           | 100.99      | 9        | false      |
```

Comment:
Why the final price is changing ? Impact on price are always there.

Reply:
If this column is not tested, Behat should send a :red_circle: alert.

Reply:
It depends how the test are built, if we only test the values we expect then it can't guess it alone

But anyway this field doesn't exist anymore in the QueryResult, and the formerly associated test has been removed as well This is just a relicate of former code, all these columns should be removed from feature files

Reply:
right final price is not used anymore, because it will be calculated in javascript based on price + impact on price :+1: